src/components/CvTooltip/CvTooltip.stories.js

```diff
@@ -0,0 +1,125 @@
+import CvTooltip from './CvTooltip.vue';
+import {
+  sbCompPrefix,
+  storyParametersObject,
+} from '../../global/storybook-utils';
+import { alignments, directions } from './consts';
+import { sbTooltipPrefix } from './sbTooltipPrefix';
+
+export default {
+  title: `${sbCompPrefix}/${sbTooltipPrefix}/CvTooltip`,
+  component: CvTooltip,
+  argTypes: {
+    alignment: {
+      control: {
+        type: 'select',
+      },
+      options: alignments,
+      defaultValue: CvTooltip.props.alignment.default,
+    },
+    direction: {
+      control: {
+        type: 'select',
+      },
+      options: directions,
+      defaultValue: CvTooltip.props.direction.default,
+    },
+    default: {
+      table: {
+        type: { summary: 'string | html | Component' },
+        category: 'slots',
+      },
+      description: 'Tooltip trigger slot.',
+    },
+    content: {
+      table: {
+        type: { summary: 'string | html | Component' },
+        category: 'slots',
+      },
+      description: 'Tooltip text slot',
+    },
+  },
+};
+
+const TipText = 'This is a tooltip';
+
+const template = `<cv-tooltip v-bind="args" />`;
+
+const Template = args => {
+  return {
+    components: { CvTooltip },
+    template,
+    setup() {
+      return {
+        args,
+      };
+    },
+  };
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  tip: TipText,
+};
+Default.parameters = storyParametersObject(
+  Default.parameters,
+  template,
+  Default.args
+);
+
+const triggerSlotTemplate = `
+  <cv-tooltip v-bind="args">
+    <svg width="16" height="12" viewBox="0 0 16 12">
+      <path d="M8.05 2a2.5 2.5 0 0 1 4.9 0H16v1h-3.05a2.5 2.5 0 0 1-4.9 0H0V2h8.05zm2.45 2a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3zM3.05 9a2.5 2.5 0 0 1 4.9 0H16v1H7.95a2.5 2.5 0 0 1-4.9 0H0V9h3.05zm2.45 2a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3z" />
+    </svg>
+  </cv-tooltip>
+`;
+
+const TriggerSlotTemplate = args => {
+  return {
+    components: { CvTooltip },
+    template: triggerSlotTemplate,
+    setup() {
+      return {
+        args,
+      };
+    },
+  };
+};
+
+export const TriggerSlot = TriggerSlotTemplate.bind({});
+TriggerSlot.parameters = storyParametersObject(
```

Comment:
This is the only one without a default tooltip text so it looks like this when I hover 
![image](https://github.com/carbon-design-system/carbon-components-vue/assets/7536103/40b3129b-93d8-4e28-9145-52da5097e9d9)
```diff
diff --git a/src/components/CvTooltip/CvTooltip.stories.js b/src/components/CvTooltip/CvTooltip.stories.js
index e31778ae..8d538919 100644
--- a/src/components/CvTooltip/CvTooltip.stories.js
+++ b/src/components/CvTooltip/CvTooltip.stories.js
@@ -88,6 +88,9 @@ const TriggerSlotTemplate = args => {
 };
 
 export const TriggerSlot = TriggerSlotTemplate.bind({});
+TriggerSlot.args = {
+  tip: TipText,
+};
 TriggerSlot.parameters = storyParametersObject(
   TriggerSlot.parameters,
   triggerSlotTemplate,
```